programs/bucket-program/src/state/oracle/switchboard.rs

```diff
@@ -0,0 +1,42 @@
+use {
+    crate::{
+        error::ErrorCode,
+        math::casting::{cast_to_i64, cast_to_u128},
+        math_error,
+    },
+    anchor_lang::prelude::*,
+    switchboard_v2::AggregatorAccountData
+};
+
+pub fn get_switchboard_price(
+    switchboard_feed_info: &AccountInfo,
+    clock_slot: u64,
+    target_precision: u32,
+) -> Result<(i128, i128, u128, i64), ErrorCode> {
+    let feed_result = AggregatorAccountData::new(switchboard_feed_info).unwrap();
+
+    let switchboard_decimal = AggregatorAccountData::new(switchboard_feed_info).unwrap().get_result().unwrap();
+
+    let oracle_price: i128 = switchboard_decimal.mantissa.checked_div(10_i128.checked_pow(switchboard_decimal.scale).unwrap()).unwrap();
+
+    let std_dev = feed_result.latest_confirmed_round.std_deviation;
```

Comment:
can you add explicit type definition on `std_dev`?

Reply:
outdated - new code from drift protocol